assembl/models/auth.py

```diff
@@ -1682,6 +1693,7 @@ def getCurrent(cls, req=None):
             req = get_current_request()
         assert req
         if getattr(req, "lang_prefs", 0) is 0:
+            # lang. prefs remain effective even without shared login
             user_id = req.authenticated_userid
```

Comment:
while you're at it, use `authenticated_userid(req)`
The syntax req.authenticated_userid is deprecated.